### PR TITLE
[logging] Add more flags to default logs

### PR DESCRIPTION
**Pull request by anijain2305**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #107912

### CODE REVIEW

**Review by mlazos — APPROVED** [review]

File: torch/_logging/_internal.py [hunk 1]
```diff
@@ -120,6 +120,8 @@ def clear(self):
 
 # sample usage: torch._logging.set_logs(**torch._logging.DEFAULT_LOGGING)
 DEFAULT_LOGGING = {
+    "graph_code": True,
+    "aot": logging.INFO,
```
Comment (ezyang) [inline torch/_logging/_internal.py:124]: Actually don't we want dynamo logging.INFO too?

Comment (anijain2305) [reply torch/_logging/_internal.py:125]: Added

**Pull request closed without merging**